### DISCUSSION TITLE
Wiki Github branch naming 'master' versus 'main'

### DISCUSSION THREAD

**Discussion by adenoz** [requarks/wiki#3247] [Error / Bug Report] [answered]
I have recently setup wiki.js and just finished getting git and Github setup as the storage option. I noticed after successfully setting up Github and getting the first syncs going that the wiki.js data was syncing to a Github branch 'master', which I named as [per the instructions](https://docs.requarks.io/storage/git), under the 'Configure Wiki.js' section. However, I think the instructions to name the branch 'master' is now dated. [Github has recently](https://github.com/github/renaming) moved away from naming the default repos from 'master' to 'main'.

What this meant for me was that my wiki.js repo had two branches, the default 'main' (because it is a new repo) and the new wiki.js created one called 'master'. And it is not possible to merge them easily because there is no common commit history.

There are some fairly simple fixes like changing the default branch from 'main' to 'master' etc but a better overall fix would be to just update the instructions/documentation that instructs new users on what name to use for the 'Branch' name from 'master' to 'main'. Then git syncs will just work and users will have just the one clean branch in their repo to start with.

I'd be happy to make an edit and submit a pull request for something so simple but I can't actually find the entry in Github, although I am fairly new to all of this so may have missed something obvious. For consideration.

**Comment by NGPixel** [comment, marked as answer]
Updated the doc to `main`
